students/Vincent_Aquila/session03/list_lab.py

```diff
@@ -0,0 +1,185 @@
+list.insert(0, "Strawberry")
+print(list)
+"""
+I learned a lot here - tried to use:
+    l = (str(k)[1:-1])
+which removes brackets from the list in the terminal display - but - after
+stripping brackets from a list and then employing an insert method -
+it won't work - throws an attribute error.  Insert only works with a bracketed
+list.  Unable to use the + sign to add the second fruit to the beginning of the
+list.
+"""
+
+#add a thrid fruit to the beginning of the list using insert()
+list.insert(0, "Kiwi")
+print(list)
+
+#display all the friuts that begin with "P" using a for loop
+for item in list:
+	if "P" in item:
+		print(item)
+print("---End of Series 1 Exercises---")
+#end of Series 1
+
+
+
+#Series 2
+"""
+Using the list created in series 1 above:
+-Display the list.
+-Remove the last fruit from the list.
+-Display the list.
+-Ask the user for a fruit to delete, find it and delete it.
+(Bonus: Multiply the list times two. Keep asking until a match is found. Once
+found, delete all occurrences.)
+"""
+
+print(list)
+list2 = list
+#remove the last fruit from the list and display the new list
+list2.pop()
+print(list2)
+
+#ask the user for a fruit to delete and delete it
+response = input("Select a fruit to delete from the list. > ")
+response = response.title()
+list2.remove(response)
+print(list2)
+
+double_list = list2 * 2
+print(double_list)
+print("---End of Series 2 Exercises---")
+#end of Series 2
+
+
+
+#Series 3
+"""
+Again, using the list from series 1:
+-Ask the user for input displaying a line like “Do you like apples?” for each
+fruit in the list (making the fruit all lowercase).
+-For each “no”, delete that fruit from the list.
+-For any answer that is not “yes” or “no”, prompt the user to answer with one of
+those two values (a while loop is good here)
+-Display the list.
+"""
+
+list3 = ["Apples", "Pears", "Oranges", "Peaches"] #original list
+response_apples = input("Do you like apples?  Please respond with yes or no > ")
+if response_apples == "no":
+    list3.remove("Apples")
+print(list3)
+response_pears = input("Do you like pears?  Please respond with yes or no > ")
+if response_pears == "no":
+    list3.remove("Pears")
+print(list3)
+response_oranges = input("Do you like oranges?  Please respond with yes or no > ")
+if response_oranges == "no":
+    list3.remove("Oranges")
+print(list3)
+response_peaches = input("Do you like peaches?  Please respond with a yes or no > ")
+if response_peaches == "no":
+    list3.remove("Peaches")
+print(list3)
+print("---End of Series 3 Exercies---")
+#end of Series 3
+
+
+
+"""
+Series 4
+Once more, using the list from series 1:
+-Make a copy of the list and reverse the letters in each fruit in the copy.
+-Delete the last item of the original list. Display the original list and the copy.
+"""
+
+list = ["Apples", "Pears", "Oranges", "Peaches"]
+list4 = list[:] #making a copy of list, now called list4
+print(list4)
+
+Apples = (list[0])
```

Comment:
again, you should be able to do this with a for loop.

Whenever you find yourself writing repetitive code -- figure out how not to!